nixos/lib/make-iso9660-image.sh

```diff
@@ -117,15 +118,6 @@ xorriso="xorriso
 
 $xorriso -output $out/iso/$isoName
 
-if test -n "$usbBootable"; then
```

Comment:
I'm not sure whether we can remove isohybrid. It could very well be that the parameters passed to `xorriso` are already sufficient, but from what I remember there were quite a few systems we needed to support. Make sure to test this on a UEFI machine, a legacy bios machine and an OSX machine.

If we're indeed removing this, we might as well remove the `usbBootable` options from `make-iso9660-image.nix`. Might be better to do this in a separate PR.

Reply:
(I've since tested this both on UEFI, non-UEFI and OSX machines. agreed to do further refactoring in a separate PR)